docs/source/app-dev/services.rst

```diff
@@ -0,0 +1,159 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+The Ledger API services
+#######################
+
+
+The Ledger API is structured as a set of services. The core services are implemented using `gRPC <https://grpc.io/>`__ and `Protobuf <https://developers.google.com/protocol-buffers/>`__, but most applications will not use this low-level API, instead using the language bindings (e.g. `Java <#java>`__, `Javascript <#javascript>`__).
+
+This page gives more detail about each of the services in the API, and will be relevant whichever way you're accessing it.
+
+If you want to read low-level detail about each service, see the :doc:`protobuf documentation of the API <grpc/proto-docs>`.
+
+Overview
+********
+
+The API is structured as two separate data streams:
+
+-  A stream of **commands** TO the ledger that allow an application to cause state changes.
+-  A stream of **events** FROM the ledger that indicate all state changes taking place on the ledger.
```

Comment:
```suggestion
-  A stream of **transactions** and corresponding **events** FROM the ledger that indicate all state changes that have taken place on the ledger.
```

Reply:
The key thing here seems to be events, not transactions. For example, it's difficult to interpret the ACS or command completion service as delivering "transactions".